onnxruntime/core/session/standalone_op_invoker.cc

```diff
@@ -94,13 +91,13 @@ class NodeRepo {
     return node_repo;
   }
 
-  onnxruntime::Status AddNode(const onnxruntime::OpKernel* kernel, NodePtr node_ptr) {
+  onnxruntime::Status AddNode(const onnxruntime::OpKernel* kernel, NodePtr&& node_ptr, ArgPtrs&& args) {
     std::lock_guard<std::mutex> guard(mutex_);
-    auto iter = node_map_.find(kernel);
-    if (iter != node_map_.end()) {
+    auto iter = resource_map_.find(kernel);
+    if (iter != resource_map_.end()) {
       return ORT_MAKE_STATUS(ONNXRUNTIME, INVALID_ARGUMENT, "kernel already mapped to existing node");
     }
-    node_map_.insert({kernel, std::move(node_ptr)});
+    resource_map_[kernel] = {std::forward<NodePtr>(node_ptr), std::forward<ArgPtrs>(args)};
```

Comment:
nit: prefer insert to prevent the subscript operator from creating a default constructed value.

Reply:
also std::move() is more idiomatic here since we are not dealing with forwarding references (T&& where T is a template parameter).
https://stackoverflow.com/a/28828689

Reply:
Let's go with move then, thanks!